media-sound/denemo/denemo-2.2.0.ebuild

```diff
@@ -0,0 +1,126 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit xdg-utils
+
+DESCRIPTION="A music notation editor"
+HOMEPAGE="http://www.denemo.org/"
+SRC_URI="mirror://gnu/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3 OFL-1.1"
```

Comment:
Could you verify whether the copyright notice in source files does not indicate `GPL version 3 (...) or newer`? If so, please use `GPL-3+` instead.

Reply:
Hmm, some of the source files, i.e. include/denemo/denemo_types.h indicate even GPL-2+, others, like src/source/source.h indicate GPL-3+
Should I add GPL-2+ and change GPL-3+ or would only GPL-2+ suffice in that case?

Reply:
Changed to GPL-2+ for now. Just tell me, if this is not the right decision.

Reply:
GPL-2+ = GPL-2 || GPL-3 || GPL-4...
    GPL-3+ = GPL-3 || GPL-4...

So generally you can't use the whole thing under GPL-2. You can either list both explicitly if you'd like to be precise, or just make it GPL-3+ since it covers both.